DigitalLearningSolutions.Web/ViewModels/Supervisor/MyStaffListViewModel.cs

```diff
@@ -20,6 +22,7 @@ public class MyStaffListViewModel : BaseSearchablePageViewModel
             int page
         ) : base(searchString, page, false, sortBy, sortDirection, searchLabel: "Search administrators")
         {
+            IsNominatedSupervisor = adminUser?.IsNominatedSupervisor ?? false;

```

Comment:
Given that the adminUser may, in theory, be a Supervisor as well as a NominatedSupervisor, we should handle that here (`IsNominatedSupervisor` should be `false` if `adminUser.IsSupervisor` is `true`).

Reply:
Added [this commit](https://github.com/TechnologyEnhancedLearning/DLSV2/pull/982/commits/0d0be3c7edd21986c22f9feb24c0abe6d74d9ba2).